pkg/api/convertertoagentpoolonlyapi.go

```diff
@@ -349,6 +350,13 @@ func convertV20180331AgentPoolOnlyWindowsProfile(obj *v20180331.WindowsProfile)
 	}
 }
 
+func convertV20180331AgentPoolOnlyKubernetesConfig(enableRBAC bool) *KubernetesConfig {
```

Comment:
@rjtsdl thanks the making the change as this also has a good side effect of resolving bug 2265988 in VSTS! You can also now remove lines 126-141 from `putmanagedcluster.go` and close out bug 2265988 in VSTS when new ACS E is vendored.

Reply:
Made the change for v20170831 to make sure those two flags to be false ptr. Then RP don't need to deal with that.